openedx/core/djangoapps/user_authn/views/tests/test_logistration.py

```diff
@@ -423,15 +423,17 @@ def test_settings_tpa_hinted_login_dialog_disabled(self, url_name, auth_entry):
 
     @mock.patch('openedx.core.djangoapps.user_authn.views.login_form.enterprise_customer_for_request')
     @ddt.data(
-        ('signin_user', False, None, None),
-        ('register_user', False, None, None),
-        ('signin_user', True, 'Fake EC', 'http://logo.com/logo.jpg'),
-        ('register_user', True, 'Fake EC', 'http://logo.com/logo.jpg'),
-        ('signin_user', True, 'Fake EC', None),
-        ('register_user', True, 'Fake EC', None),
+        ('signin_user', False, None, None, False),
+        ('register_user', False, None, None, False),
+        ('signin_user', True, 'Fake EC', 'http://logo.com/logo.jpg', False),
+        ('register_user', True, 'Fake EC', 'http://logo.com/logo.jpg', False),
+        ('signin_user', True, 'Fake EC', 'http://logo.com/logo.jpg', True),
+        ('register_user', True, 'Fake EC', 'http://logo.com/logo.jpg', True),
+        ('signin_user', True, 'Fake EC', None, False),
+        ('register_user', True, 'Fake EC', None, False),
```

Comment:
I know you didn't write this, but this feels like a good place to use the ddt dictionary unpacking for readability

Reply:
I wrestled with this for a while, following examples in our codebase as well as the examples provided by ddt docs. I couldn't get it to work seemingly because of the mock.patch, not worth spending any more time on IMO.

Reply:
Ah I ran into this, mock.patch needs to be first here I think

Reply:
Don’t worry for this PR though